main_test.go

```diff
@@ -17,15 +17,27 @@ func TestMain(m *testing.M) {
 	}))
 }
 
+func commonEnv(t *testing.T) func(env *testscript.Env) error {
+	return func(env *testscript.Env) error {
+		env.Setenv(string(v2.EnvDev), "true")
+		env.Setenv("HOME", "$WORK/home")
+		env.Setenv("VERSION", static.Version)
+		env.Setenv("COMMIT_SHA", static.Sha)
+		return nil
+	}
+}
 func TestScripts(t *testing.T) {
+	t.Parallel()
+	testscript.Run(t, testscript.Params{
+		Dir:   "testdata/scripts",
+		Setup: commonEnv(t),
+	})
+}
+
+func TestNodeDbScripts(t *testing.T) {
+	t.Parallel()
 	testscript.Run(t, testscript.Params{
-		Dir: "testdata/scripts",
-		Setup: func(env *testscript.Env) error {
-			env.Setenv(string(v2.EnvDev), "true")
-			env.Setenv("HOME", "$WORK/home")
-			env.Setenv("VERSION", static.Version)
-			env.Setenv("COMMIT_SHA", static.Sha)
-			return nil
-		},
+		Dir:   "testdata/scripts/node/db",
```

Comment:
Did you look for any way around this? Like a `**/*` wildcard? I almost wrote a wrapper to walk the file tree in my first PR but bailed when it seemed messy.

Reply:
i looked at the package to see if there was a way to filter or match files but didn't find anything.

i prefer this organization for a few reasons:
1. primarily, it much easier to debug because testscript boilerplate to dig thru
2. it's easy to think about if we organize the txtar dirs like we do the cmds themselves
3. it's faster because we can parallel.


this was just a first step to reorganizing, which i'm happy to do in a other PR

i tried added my test and got totally lost in the output, which motivated me to do this and goes back to pt #1

Reply:
Yeah I like the organization, but the Test names become unnecessarily complicated with separate top level `func Test`s for each sub directory, and devs have to know/remember to add new ones. I think we can do something like this tough:
```go
//go:embed testdata/scripts
var scripts embed.FS

func TestScripts(t *testing.T) {
	t.Parallel()
	fs.WalkDir(scripts, "testdata/scripts",  func(path string, e fs.DirEntry, err error) error {
		if !e.IDir() { return nil }
		dir := e.Name()
		t.Run(dir, func(t *testing.T) {
			t.Parallel()
			testscript.Run(t, testscript.Params{
				Dir:   dir,
				Setup: commonEnv(t),
			})
		})
	})
}
```

Reply:
Some version of that ^ could give us all the benefits, but with clean TestScripts/cmd/subcmd naming